compile.sh

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
-cp /app/code/* .
-DreamMaker test.dme
-DreamDaemon test.dmb -close -verbose -ultrasafe | cat
```

Comment:
What's the benefit to deleting this? If you remove this you'll have to alter the entry point as well.